wiki/Skinning/Sounds/en.md

```diff
@@ -515,6 +515,8 @@ Notes:
 Notes:
 
 - Failing a map at any point.
+- Not intended to last for longer than the fail animation (about 5s).
+- Using a long audio track will impact performance as it is loaded each time you play a beatmap, even if you never fail. It's not supposed to be bigger than a few kilobytes.
```

Comment:
```suggestion
- Using a long audio track will impact game client's performance, as it is loaded each time you play a beatmap, even if you never fail. It's not supposed to be bigger than a few kilobytes.
```

Reply:
Using a long audio track will impact the game client's performance, as it is loaded each time you play a beatmap, even if you never fail. It's not supposed to be bigger than a few kilobytes.

Reply:
@circleen try using the "insert a suggestion" button or wrap your suggestion in a code block next time (I edited it for you here)

Reply:
sorry, didn't see the button there.